theano/tensor/blas_headers.py

```diff
@@ -984,7 +983,11 @@ def blas_header_text():
                     }
                     """)
 
-    return (header % {'const': const}) + gemm_code
+    return (header % {'const': const}) + blas_code
+
+
+if not config.blas.ldflags:
+    print('Using NumPy based implementation for BLAS functions.')
```

Comment:
Using NumPy C-API based ... So make it more clear that we still use C code.

But I'm not sure we should add this print expecially if on windows we don't link with anaconda mkl packages.

@lamblin what do you think?